messages/tape_linux_sg/root.txt

```diff
@@ -100,6 +100,7 @@ root:table {
 		30258W:string { "Cannot retrieve drive dump: failed to read buffer (%d)." }
 		30259W:string { "Cannot retrieve drive dump: failed to write to dump file (%d)." }
 		30260W:string { "Cannot retrieve drive dump: wrote %d bytes out, expected %d." }
+		30261W:string { "Cannot retrieve drive dump: failed to communicate with drive. Tried (%d) times." }
```

Comment:
The number need to unique. In other words, `30261W` and `30261I` cannot be defined at one time.

I think `30296` is the good number.